test/SourceKit/CursorInfo/invalid_offset.swift

```diff
@@ -12,4 +12,4 @@ let a = 12
 // rdar://problem/38162017
 // REQUIRES: OS=macosx
 
-// CHECK: <empty cursor info>
+// CHECK: (Request Failed): Unable to resolve the start of the token
```

Comment:
@rintaro, in your fix for rdar://30346106 you had the request return an empty response for out-of-range offset, are you ok with the response failing instead ?

Reply:
I believe it should be OK.